orm/src/test/java/org/hibernate/search/test/batchindexing/DatabaseMultitenancyTest.java

```diff
@@ -203,20 +200,20 @@ public void shouldSearchOtherTenantsDocuments() throws Exception {
 	private void rebuildIndexWithMassIndexer(Class<?> entityType, String tenantId) throws Exception {
 		FullTextSession session = Search.getFullTextSession( openSessionWithTenantId( tenantId ) );
 		session.createIndexer( entityType ).purgeAllOnStart( true ).startAndWait();
-		IndexReader indexReader = session.getSearchFactory().getIndexReaderAccessor().open( entityType );
-		long tenantIdTermFreq = indexReader.docFreq( new Term( DocumentBuilderIndexedEntity.TENANT_ID_FIELDNAME, tenantId ) );
+//		IndexReader indexReader = session.getSearchFactory().getIndexReaderAccessor().open( entityType );
```

Comment:
Just seeing these, I'll re-work that one. Should be easily doable with the more abstract test helper I've created.